trunk/includes/pardot-api-class.php

```diff
@@ -60,74 +62,39 @@ class Pardot_API {
 	private static $LOGIN_URL_PATH_TEMPLATE = '/login/version/%%VERSION%%';
 
 	/**
-	 * @var string A user entered email address that is expected to have a valid Pardot account.
+	 * @var string A user-entered client ID.
 	 *
-	 * @since 1.0.0
+	 * @since 1.5.0
 	 */
-	var $email;
+	var $client_id;
 
 	/**
-	 * @var string A user-entered password for the valid Pardot account indentified by $this->email.
+	 * @var string A user-entered client secret.
 	 *
-	 * @since 1.0.0
+	 * @since 1.5.0
 	 */
-	var $password;
+	var $client_secret;
 
 	/**
-	 * @var string A user-entered but Pardot-supplied key the user accesses from their Pardot account.
+	 * @var string A user-entered business unit ID.
 	 *
-	 * @since 1.0.0
+	 * @since 1.5.0
 	 */
-	var $user_key;
-
-    /**
-     * @var string The type of authentication, either Pardot or SSO.
-     *
-     * @since 1.5.0
-     */
-    var $auth_type;
-
```

Comment:
These were used for Pardot authentication, which we no longer support and are removing the option in this update.